src/bootstrap/dist.rs

```diff
@@ -456,7 +456,7 @@ impl Step for DebuggerScripts {
     type Output = ();
 
     fn should_run(run: ShouldRun<'_>) -> ShouldRun<'_> {
-        run.path("src/etc/lldb_batchmode.py")
+        run.never()
     }
 
     fn make_run(run: RunConfig<'_>) {
```

Comment:
We can delete make_run if should_run is run.never() I believe.

Reply:
It took me a while to grasp what this would mean but I think I agree. I've removed the function in a fixup. The default implementation should suffice.